src/redux/eventSlice.js

```diff
@@ -0,0 +1,114 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+export const getEventAsync = createAsyncThunk(
+  'events/getEventAsync',
```

Comment:
If your reducer is singular (`event`) keep it in here at singular as well - or change it to be plural everywhere.

Reply:
done